Sources/Parsers/AssetsCatalogParser.swift

```diff
@@ -104,6 +107,9 @@ extension AssetsCatalogParser {
     let type = item.extension ?? ""
 
     switch (type, AssetCatalog.Item.supported.contains(type)) {
```

Comment:
Good catch, that should have been `case ("", _), (_, false):`. I think that was a remainder from when we had a blacklist. Xcode allows group names with `.` in them. Groups with extensions that match an Xcode set extension are considered of that set type.

We have 2 options:
- maintain a list of all asset catalog types, and have a supported & unsupported list so we can correctly match actual sets <-> groups.
- ignore all folders with "extensions" in them that we don't support, potentially erroneously thinking that a group is an unsupported set type.